### PR TITLE
Add provider for searching IETF RFCs and Drafts

### PR DESCRIPTION
**Pull request by akb**
`s -p ietf websocket`

### CODE REVIEW

**Comment by KeizerDev** [comment]
Please stick to the naming conventions. `IETF` has to be `IETFProvider`

File: providers/ietf/ietf.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+func init() {
+	providers.AddProvider("ietf", &IETF{})
+}
+
+// IETF adheres to the Provider interface.
+type IETF struct {
+}
+
+// BuildURI generates a search URL for IETF.
+func (p *IETF) BuildURI(q string) string {
```
Comment (KeizerDev) [inline providers/ietf/ietf.go]: Please stick to the naming conventions. `IETF` has to be `IETFProvider`

File: providers/ietf/ietf.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("ietf", &IETF{})
+}
+
+// IETF adheres to the Provider interface.
+type IETF struct {
```
Comment (KeizerDev) [inline providers/ietf/ietf.go]: Please stick to the naming conventions. `IETF` has to be `IETFProvider`

File: providers/ietf/ietf.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("ietf", &IETF{})
```
Comment (KeizerDev) [inline providers/ietf/ietf.go]: Please stick to the naming conventions. `IETF` has to be `IETFProvider`

**Comment by KeizerDev** [comment]
When fixed, please only add just one single commit, not multiple.

**Comment by KeizerDev** [comment]
@akb Please add it to the README.md too.

File: providers/ietf/ietf.go (hunk 3 above)
Comment (akb) [inline providers/ietf/ietf.go]: Understood. I used the Steam provider as a template which also does not follow the convention. I will submit a separate PR to correct it.

**Comment by akb** [comment]
You got it, boss.

**Comment by zquestz** [comment]
If you want feel free to fix Steam =)

**Pull request merged**